save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestExecutionService.kt

```diff
@@ -51,17 +51,37 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
      * @param page a zero-based index of page of data
      * @param pageSize size of page
      * @param status
+     * @param testName
      * @param testSuite
+     * @param tag
      * @return a list of [TestExecutionDto]s
      */
-    @Suppress("AVOID_NULL_CHECKS", "UnsafeCallOnNullableType")
+    @Suppress("AVOID_NULL_CHECKS", "UnsafeCallOnNullableType", "TOO_MANY_PARAMETERS")
     internal fun getTestExecutions(
         executionId: Long,
         page: Int,
         pageSize: Int,
         status: TestResultStatus?,
+        testName: String?,
         testSuite: String?,
```

Comment:
`TestSuite` is an entity, but you mean `testSuiteName` here. Pls rename

Reply:
Done